examples/src/main/kotlin/io/aexp/bucketlist/examples/prlifetime/ExportPrLifetimeData.kt

```diff
@@ -39,17 +39,22 @@ import java.util.concurrent.TimeUnit
  *
  * Usage:
  * - make a properties file containing 'username', 'password', and 'url' info for your Bitbucket-Server instance.
- * - invoke with <path to props file> <project key> <repo slug> <output file> <start date> <end date>
+ * - invoke with <path to props file> <project key> <repo slug> <output file> <start date> <end date> <start duration from>
  * - dates should be in the format YYYY-MM-DD
  */
 object ExportPrLifetimeData {
 
     val logger: Logger = LoggerFactory.getLogger(javaClass)
 
+    enum class DurationStart {
+        prCreation,
+        lastPrCommitPushed
+    }
+
     @JvmStatic fun main(args: Array<String>) {
 
-        if (args.size != 6) {
-            System.err!!.println("Must have 6 arguments: <config file> <project key> <repo slug> <output file> <start date> <end date>")
+        if (args.size != 7) {
+            System.err!!.println("Must have 6 arguments: <config file> <project key> <repo slug> <output file> <start date> <end date> <start duration from>")
```

Comment:
error message need to be updated to reflect new list. Also, should extract that to a val.